ksql-serde/src/main/java/io/confluent/ksql/serde/json/KsqlJsonDeserializer.java

```diff
@@ -75,12 +75,15 @@
       .build();
 
   private final PersistenceSchema physicalSchema;
+  private final boolean isJsonSchema;
   private String target = "?";
 
   public KsqlJsonDeserializer(
-      final PersistenceSchema physicalSchema
+      final PersistenceSchema physicalSchema,
+      final boolean isJsonSchema
   ) {
     this.physicalSchema = JsonSerdeUtils.validateSchema(physicalSchema);
```

Comment:
Null check here?

Reply:
I've got another PR following up this one to fix loose ends and will make sure to add it there! (I want to get this in since the build is green)

Thanks for the review @vpapavas!